buku

```diff
@@ -2511,61 +2516,70 @@ class BukuDb:
         resp = 'y'
 
         try:
-            if self.chatty:
-                resp = input('Import bookmarks from google chrome? (y/n): ')
-            if resp == 'y':
-                bookmarks_database = os.path.expanduser(gc_bm_db_path)
-                if not os.path.exists(bookmarks_database):
-                    raise FileNotFoundError
-                self.load_chrome_database(bookmarks_database, newtag, add_parent_folder_as_tag)
+            if os.path.isfile(os.path.expanduser(gc_bm_db_path)):
```

Comment:
Adding a file existence check to avoid asking about nonexistent bookmark files.
(`expanduser()` is required to handle `~` in paths)